packages/backend-modules/mail/templates/subscription_upgrade_activated_BENEFACTOR.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="x-ua-compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <!--[if gte mso 15]>
+      <xml>
+        <o:officedocumentsettings>
+          <o:allowpng />
+          <o:pixelsperinch>96</o:pixelsperinch>
+        </o:officedocumentsettings>
+      </xml>
+    <![endif]-->
+    <style type="text/css">
+      {{{sg_font_faces}}}
+    </style>
+    <style type="text/css">
+      p {
+        color:#282828;font-size:17px;line-height:24px;
+        {{{sg_font_style_sans_serif_regular}}}
+      }
+      p strong {
+        {{{sg_font_style_sans_serif_medium}}}
+      }
+    </style>
+  </head>
+  <body style="margin: 0; padding: 0; background-color: #fff">
+    <table border="0" cellpadding="0" cellspacing="0" width="100%">
+      <tbody>
+        <tr>
+          <td align="center" valign="top">
+            <table
+              align="center"
+              border="0"
+              cellpadding="0"
+              cellspacing="0"
+              width="100%"
+              style="max-width:640px;color:#282828;font-size:17px;line-height:24px;{{sg_font_style_sans_serif_regular}}"
+            >
+              <tbody>
+                <tr>
+                  <td style="padding: 20px">
+                    <p>Guten Tag</p>
+                    <p>
+                      Wir freuen uns, dass Sie sich für einen Wechsel von einem Monatsabo zur Gönnermitgliedschaft entschieden haben.
+                      Sie erhöhen dadurch unsere Planungssicherheit und sichern die Zukunft der Republik langfristig.
+                    </p>
+                    <p>Wir freuen uns, dass Sie sich für eine {{next_abo_name}} entschieden haben.</p>
+                    <p>Ihre Jahresmitgliedschaft wird am wird am {{start_date}} aktiv.</p>
+
+                    <p>
```

Comment:
Seems to be the same text as the subscription_setup_successful_upgrade mail, but I guess this will still be adapted